README.md

```diff
@@ -8,152 +8,225 @@
-Server, and that's a useful way to incrementally adopt containerization.
-
-The toolbox environment is based on an [OCI](https://www.opencontainers.org/)
-image. On Fedora this is the `fedora-toolbox` image. This image is used to
-create a toolbox container that seamlessly integrates with the rest of the
-operating system.
-
-## Usage
+## Basic usage
 
 ### Create your toolbox container:
-```
+
+```shell
 [user@hostname ~]$ toolbox create
-Created container: fedora-toolbox-30
+Created container: fedora-toolbox-32
 Enter with: toolbox enter
 [user@hostname ~]$
 ```
-This will create a container called `fedora-toolbox-<version-id>`.
+
+This will create a container called `fedora-toolbox-<version-id>` based on the
+version of Fedora you are using.
 
 ### Enter the toolbox:
-```
+
+```shell
 [user@hostname ~]$ toolbox enter
 ⬢[user@toolbox ~]$
 ```
 
-## Dependencies and Installation
+> Notice: The '⬢' symbol may not be present if you are using a custom shell
+> theme.
+
+### Remove a toolbox container:
+
+```shell
+[user@hostname ~]$ toolbox rm <container-name>
+[user@hostname ~]$
+```
+
+## Tell me more about Toolbox!
+
+Operating systems that are based on [OSTree](https://ostree.readthedocs.io/en/latest/)
+like [CoreOS](https://coreos.fedoraproject.org/) and
+[Fedora Silverblue](https://silverblue.fedoraproject.org/) discourage
+installation of software on the host (they mostly don't even have a *classic*
+package manager), and instead install software as/in containers.
+
+**Hmm, why is that so?**
+
+There are several advantages to this approach including higher reliability,
+easier upgrading and easier debugging. Fedora Magazine has nice articles about
+both [CoreOS](https://fedoramagazine.org/introducing-fedora-coreos/) and
+[Silverblue](https://fedoramagazine.org/what-is-silverblue/).
+
+**OK, what is then Toolbox good for?**
+
+Toolbox is a tool built on top of [Podman](https://podman.io) (which is a tool
+for managing [OCI](https://www.opencontainers.org/)-based containers) providing
+sane default values when creating/running containers (so-called *toolboxes*).
+These containers then *seamlessly integrate* with the rest of the operating
+system (e.g., environmental variables, home folder, shared memory, important
+locations in the root filesystem).
+
+Bear in might that Toolbox is not meant to be "safe" in terms of isolating your
+actions in the container from the host. It strives to extend your workflow
+rather than isolate it using the container technology.
+
+**Uh.., what?**
+
+Let's say you want to try to use `ansible` but you don't want to bloat your
+system (e.g., Fedora Workstation) or you can't really install it comfortably
+(e.g., Silverblue).
+
+Just create a container with `toolbox create`, enter it with `toolbox enter`
+(you should see the '⬢' symbol at the beginning of the command prompt) and
+type `sudo dnf install ansible` (the container is Fedora-based).
```

Comment:
One good thing about having the example commands at the top of the `README.md` is that we don't need to describe the steps in detail. :)